crates/telio-model/src/features.rs

```diff
@@ -690,6 +698,7 @@ mod tests {
             enable_nat_type_collection: false,
             enable_relay_conn_data: true,
             enable_nat_traversal_conn_data: true,
+            state_duration_cap: 60 * 60 * 24,
```

Comment:
Why not also use default_state_duration_cap() here? Less manual work if we needed to change the value in the future.